aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -80,6 +80,9 @@ pub fn allow_module_bundle_for_test() {
     MODULE_BUNDLE_DISALLOWED.store(false, Ordering::Relaxed);
 }
 
+// Put a hard_limit on per item size, 4MB.
+static MAX_ITEM_SIZE_ALLOWED: usize = 4 << 20;
```

Comment:
1MB is already "too big" in my book, what's the the minimal limit that will still allow any valid use case in your opinion? @wrwg @vgao1996 @davidiw ?

Also.. relaxing this will be way easier than making it stricter in the future if needed.

Reply:
I think the 4MB is likely fine, and 1MB would also do. The aptos framework with code and metadata is currently around 1MB in size, but publishing it results in multiple writes (one for code, on for data), each of them roughly 1/2 MB.

Reply:
I think the code is divided into multiple modules as well? So we should not have a single piece of code that's several hundred KBs?